### PR TITLE
JC-2378 Fixed: new uploaded avatar i not shown

### PR DESCRIPTION
**Pull request by mihnayan**
Added version of user profile in URL of avatar
for showing new uploaded avatar.

### CODE REVIEW

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java [hunk 1]
```diff
@@ -248,6 +245,21 @@ public UserProfileDto getUserProfileDto() {
...
+     * Set user profile version
+     * @param version - profile version
+     */
+    public void setUserProfileVersion(Long version) {
+        this.userProfileVersion = version;
+    }
+
+    /**
+     * @return user profile version
+     */
```
Comment (usharik) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: JavaDoc is redundant

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java [hunk 2]
```diff
@@ -248,6 +245,21 @@ public UserProfileDto getUserProfileDto() {
     }
 
     /**
+     * Set user profile version
+     * @param version - profile version
+     */
+    public void setUserProfileVersion(Long version) {
```
Comment (usharik) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: JavaDoc is redundant

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java (hunk 1 above)
Comment (mihnayan) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: I don't think so. This is a common code style. Look, every getter and setter has its own Javadoc.

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java (hunk 2 above)
Comment (mihnayan) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: I don't think so. This is a common code style. Look, every getter and setter has its own Javadoc.

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java (hunk 1 above)
Comment (usharik) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: Which useful information this JavaDoc gives us?

**Pull request closed without merging**